conjur-openapi.yml

```diff
@@ -314,6 +340,28 @@ security:
   - conjurAuth: []
   
 paths:
+  '/authenticators':
+    get:
+      tags:
+      - "status"
+      summary: "Details about which authenticators are on the Conjur Server"
+      description: |
+        "Response contains three members: installed, configured, and enabled. which tell
+        you which authenticators the Conjur server has the ability to authenticate with and
+        which ones are installed but not enabled for authentication"
```

Comment:
@orenbm is this how you would word the summary / description for the authenticator status API? would you propose any changes to how we've worded it here?